app/helper/PushNotification.js

```diff
@@ -0,0 +1,47 @@
+import { Permissions, Notifications } from 'expo';
+
+const PUSH_ENDPOINT = 'http://stanforddaily2.staging.wpengine.com/wp-json/tsd/v1/push-notification/users';
```

Comment:
todo: use a constant for this, once we deploy the push notification functionality to prod?